lua/base46/themes/gruvbox.lua

```diff
@@ -52,7 +55,23 @@ M.base_16 = {
   base0F = "#d65d0e",
 }
 
-M.type = "dark"
```

Comment:
idk why it shows as a change even though this is how this file looks like on dev branch